src/FetchWrapper.ts

```diff
@@ -1460,4 +1460,20 @@ export default class FetchWrapper {
       let url = `${this.instancesAPIURL}/env/langs/${locale}?action=${action}`
       return await this.makeRequest(url, "PUT")
    }
+
+   async appendURI(uri: string, zuid: string) {
```

Comment:
For audit trail in the homepage, i need to set specifically the instance zuid because if not it will be undefined, and I won't get the audit trail of some zuid because the current way is getting only in the cookie (ZESTY_WORKING_INSTANCE) which will be always the 1 same zuid. 

I make the function when dont have zuid pass it will get what is the automatically set in the instance zuid...